src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java

```diff
@@ -95,21 +84,21 @@ void setHelper(Ghprb helper) {
     }
 
     @Override
-    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, final BuildListener listener) throws InterruptedException, IOException {
-        logger = listener.getLogger();
```

Comment:
If you leave this line in you can revert a number of the changes below, for what it's worth.